cmd/flowrunner/testdata/flows/airtime.json

```diff
@@ -0,0 +1,51 @@
+[
+    {
+        "type": "flow",
+        "url": "http://testserver/assets/flow/8ca44c09-791d-453a-9799-a70dd3303306",
+        "content": {
+            "uuid": "8ca44c09-791d-453a-9799-a70dd3303306",
+            "name": "Airtime Test",
+            "language": "eng",
+            "localization": {},
+            "nodes": [
+                {
+                    "uuid": "75656148-9e8b-4611-82c0-7ff4b55fb44a",
+                    "actions": [{
+                        "type": "transfer_airtime",
+                        "uuid": "8720f157-ca1c-432f-9c0b-2014ddc77094",
+                        "amounts": {
+                            "BR": 2.5,
+                            "EC": 3
+                        }
+                    }],
+                    "router": {
+                        "type": "switch",
+                        "result_name": "Transfer",
+                        "default_exit_uuid": "",
+                        "operand": "@run",
+                        "cases": [{
+                            "uuid": "d2f852ec-7b4e-457f-ae7f-f8b243c49ff5",
+                            "type": "has_airtime_status",
+                            "arguments": ["success"],
+                            "exit_uuid": "6103fa71-6ca9-4300-aec6-929f50fa1ae0"
+                        }, {
+                            "uuid": "692926ea-09d6-4942-bd38-d266ec8d3716",
+                            "type": "has_airtime_status",
```

Comment:
I'm leaning toward keeping the names of the actions/events/tests generic - i.e. `airtime` vs `transferto`